tor/core/validation.py

```diff
@@ -75,9 +104,17 @@ def verified_posted_transcript(post, config):
     for top_level_comment in linked_resource.comments.list():
         if (
             _author_check(post, top_level_comment) and
-            _header_check(top_level_comment, config)
+            _footer_check(top_level_comment, config)
```

Comment:
I do like how we've renamed the check to, you know, not be a misnomer. `_header_check` should indicate checking the header of the comment, not the footer. 😛 

👍

Reply:
Well, it used to be the header. We just forgot to rename this when that changed.